Classes/util/ColorJSONConverter.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace csharp_08
+{
+    public class ColorJsonConverter : JsonConverter
+    {
+
+        public override bool CanConvert(Type objectType)
+        {
+            return typeof(Color) == objectType;
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            // Load token from stream
+            var token = JToken.Load(reader);
```

Comment:
Can you specify the type of the variable please, as it is not clear by reading the code